.circleci/delete-inactive-apps.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
@andrewoh531 should we use the same format/langs for creating and shutting down feature branches? E.g. per https://github.com/govau/notify/blob/master/ci/undeploy_closed_branches.py

Can discuss.

Reply:
I did it as a shell script cause it didn't look too complex and didn't want to add a dependency on Python (we'd need to be install it on the Circle CI Docker container).

If you see value in standardising I can switch.

Reply:
Happy to leave for now. We might revisit later if necessary (refer to our previous conversation about using a Makefile).